.github/ISSUE_TEMPLATE/bug.yaml

```diff
@@ -0,0 +1,71 @@
+name: Bug Report
+
+description: Report a bug in the project
+
+labels: 
+  - bug
+
+body:
+  - type: textarea
+    id: bug-description
+    attributes:
+      label: What happened?
+      description: A clear and concise description of what the bug is.
+      placeholder: Describe the bug here
+    validations:
+      required: true
+
+  - type: textarea
+    id: bug-expected-behavior
+    attributes:
+      label: What did you expect to happen?
+      description: A clear and concise description of what you expected to happen.
+    validations:
+      required: true
+
+  - type: textarea
+    id: hyperlight-version
+    attributes:
+      label: Hyperlight Version
+      description: The version of Hyperlight you are using.
+      placeholder: 0.1.0
+    validations:
+      required: true
+
+  - type: textarea
+    id: os-version
+    attributes:
+      label: OS version
+      description: The version of the OS you are using.
+      value: |
+        <details>
+
+        ```console
+        On Linux:
+        $ cat /etc/os-release
+        # paste output here
+
+        $ cat /uname -a
+        # paste output here
+
+        KVM version:
+        $ /usr/bin/qemu-system-x86_64 --version
```

Comment:
Is this really the kvm version? Idea: Maybe we can create some bash/python kind of script and check it in, that users can run to get all information we need.

Reply:
I can update to `kvm --version` but that gives the exact same output

Reply:
I think this is qemu version, not kvm

Reply:
updated

Reply:
sorry what I meant is that both are actually the qemu version

Reply:
Ah,
Should I update the description to say qemu version? or do you know a better way to get versions we want?

Reply:
There isn't a separate kvm version from the kernel version, so uname -a will capture it.